### ISSUE TITLE
When the cli receives a 5xx error, I would like to see the X-Vcap-Request-Id header value if it is present

### ISSUE THREAD

**Issue by zrob** [cloudfoundry/cli#998]
This would greatly improve the ability of operators to track down errors in logs for failed requests.

**Comment by dkoper** [comment]
@zrob Looking at a (random) trace response from CC I see this header appearing twice:
```
X-Vcap-Request-Id: 8b0c6d13-0470-4ab6-557a-683e94084160
X-Vcap-Request-Id: 8b0c6d13-0470-4ab6-557a-683e94084160::0d4a8d6e-88a1-4c9d-b0ef-392c6ab04a42
``
Do the operators need all?
Do we have documented instructions (troubleshooting guide?) on how an operator could make use of this info?

**Comment by zrob** [comment]
Hmm, i thought the double appearance was something the routing team had looked into, not sure what came of that.  The longer form is actually the shorter form plus an additional guid assigned by the CC.  Honestly either would be very helpful for troubleshooting, although ideally the longer form.

I'm not sure of the documentation of the id.

Basically an id is assigned to every request  that comes through the router.  In the case of the CC that id is used in all logs related to that request.  So that is how an operator would use it for CC.  I'm not sure how other components would use it.

**Comment by dkoper** [comment]
@shalako Do you know what came of the double appearance? (see above)

**Comment by shalako** [comment]
We don't know what the cause is of the double appearance. We did some work in april to fix a bug where the guids of the first header didn't match the first guid of the second header.

https://www.pivotaltracker.com/story/show/117866403
https://www.pivotaltracker.com/story/show/118217561

However, we apparently didn't fix the fact there are two headers. I had a story with a fairly explicit title and accepted it regardless. There must have been some context I've forgotten that is not captured in those stories.

@zrob @SocalNick and I just had a convo about this. My takeaway:
- If the CLI output the header, even if there were two of them, it would improve his team's ability to troubleshoot issues from users who aren't explicitly using the verbose option
- Zach will revisit the cause of the duplication, and work with Nick to address it if it's in CC, or send it to me if it's in the router
- CAPI will investigate leveraging support for Zipkin headers that has been added to gorouter, and adding support to CC to propogate and log these headers also

**Comment by dkoper** [comment]
Hi @zrob, @shalako 

We have implemented this in cf CLI 6.23.1, **but only for V3 and refactored commands** (i.e. task related commands and the commands listed under Refactored commands in the release notes: `api`, `delete-org`, etc.).
It will automatically apply to more commands as we refactor them (`push` is coming up).
https://github.com/cloudfoundry/cli#downloads

Regards,
Dies Koper
CF CLI PM
